test/test-debuglet.ts

```diff
@@ -436,7 +436,7 @@ describe('Debuglet', function() {
 
       it('should respect GCLOUD_DEBUG_LOGLEVEL', function(done) {
         process.env.GCLOUD_PROJECT = '11020304f2934';
-        process.env.GCLOUD_DEBUG_LOGLEVEL = 3;
+        process.env.GCLOUD_DEBUG_LOGLEVEL = '3';
```

Comment:
Nit: This shouldn't be needed because `process.env` is special in that when you set an attribute on `process.env` it will be automatically cast to a string.

Reply:
Without manually converting, it gives error as type '3' is not assigned to string|null